### ISSUE TITLE
movie generates wrong scripts if the main script ends without a newline

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#1496]
An error originally reported by @liuzhumei

![image](https://user-images.githubusercontent.com/3974108/64222292-9cf4ff80-ce9d-11e9-8ff3-8a2a9faf2097.png)

Note the `endREM` at line 16. Maybe its because the globe.bat script doesn't end with a newline?

**Comment by seisman** [comment]
Same error for bash script without an EOF newline:
```
#!/usr/bin/env bash
# Main frame loop script
export GMT_SESSION_NAME=$$
# Include static and frame-specific parameters
source movie_init.sh
source movie_params_$1.sh
mkdir ${MOVIE_NAME}
cd ${MOVIE_NAME}
gmt begin
# 	Set output PNG name and plot conversion parameters
	gmt figure ${MOVIE_NAME} png E${MOVIE_DPU},A+n+r
	gmt set PS_MEDIA 6ix6i DIR_DATA /Users/seisman/Desktop/workspace,/Users/seisman/Desktop/workspace/globe
   gmt coast -Rg -JG${MOVIE_FRAME}/20/${MOVIE_WIDTH} -Gmaroon -Sturquoise -Bg -X0 -Y0
gmt end# Move PNG file up to parent directory and cd up one level
mv -f ${MOVIE_NAME}.png ..
cd ..
# Remove frame directory and frame parameter file
rm -rf ${MOVIE_NAME}
rm -f movie_params_$1.sh

```

**Comment by PaulWessel** [comment]
Just so I understand, this is the final last line in the input script?

**Comment by seisman** [comment]
Yes, the content of the main script is:
```
gmt begin
   gmt coast -Rg -JG${MOVIE_FRAME}/20/${MOVIE_WIDTH} -Gmaroon -Sturquoise -Bg -X0 -Y0
gmt end
```
The newline character after **gmt end** make a big difference.

**Comment by seisman** [comment]
Closed by #1497.
